db-client/src/main/java/com/splicemachine/db/client/net/NetConnectionReply.java

```diff
@@ -2105,6 +2113,30 @@ private void parseACCSECRD(NetConnection netConnection, int securityMechanism) t
         }
     }
 
+    private int parsePLGINLST(NetConnection netConnection) throws DisconnectException {
+        parseLengthAndMatchCodePoint(CodePoint.PLGINLST);
+        pushLengthOnCollectionStack();
+        int peekCP = peekCodePoint();
+        if (peekCP == CodePoint.PLGINCNT) {
+            parseLengthAndMatchCodePoint(CodePoint.PLGINCNT);
+            int pluginListCount = readUnsignedShort();
+            List<String> pluginNames = new ArrayList<>(pluginListCount);
+            for (int i = 0; i < pluginListCount; i++) {
+                parseLengthAndMatchCodePoint(CodePoint.PLGINLSE);
+                parseLengthAndMatchCodePoint(CodePoint.PLGINNM);
+                pluginNames.add(readString());
+            }
+            peekCP = peekCodePoint();
+            popCollectionStack();
+            if (!pluginNames.contains(netConnection.authenticator.toUpperCase())) {
+                doSyntaxrmSemantics(CodePoint.SYNERRCD_REQ_OBJ_NOT_FOUND);
```

Comment:
can we have some log here? e.g. `"Trying to find " + netConnection.authenticator + ", but only have " + pluginNames.toString() `

Reply:
There is no usage of free text messages for agent_.logWriter_ in the code. Added more trace via .traceEntry:
`agent_.logWriter_.traceEntry(this, "parsePLGINLST", netConnection.authenticator, pluginNames.toString());`